pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -226,17 +226,19 @@ Map<String, DefaultColumnAction> computeDefaultColumnActionMap() {
         // Column exists in the segment, check if we need to update the value.
 
         if (_segmentWriter != null && !columnMetadata.isAutoGenerated()) {
-          // Check that forward index disabled isn't enabled / disabled on an existing column (not auto-generated).
-          // TODO: Add support for reloading segments when forward index disabled flag is enabled or disabled
+          // Check that forward index disabled flag isn't disabled on an existing column which had it enabled earlier
+          // (not auto-generated).
+          // TODO: Add support for reloading segments when forward index disabled flag is disabled for a segment which
```

Comment:
(nit) suggest changing this to `TODO: Add support for recreating forward index for a column with forward index disabled.` Will make it more readable imo

Reply:
done